lib_web/site.ml

```diff
@@ -17,13 +17,14 @@ type t = {
   session_backend : Sess.backend;
   router : t raw Routes.router;
   nav_links : (string * string) list;   (* Label, path *)
+  refresh_pipeline : int option;
 }
 
 class type raw_resource = [t] raw
 
 let allow_all _ _ = true
 
-let v ?(name="OCurrent") ?authn ?(secure_cookies=false) ~has_role routes =
+let v ?(name="OCurrent") ?authn ?(secure_cookies=false) ?refresh_pipeline ~has_role routes =
```

Comment:
Do we want to keep the old behaviour? In case yes, shouldn't we set it as follow:
```suggestion
let v ?(name="OCurrent") ?authn ?(secure_cookies=false) ?(refresh_pipeline=60) ~has_role routes =
```

Reply:
We cannot disable the timer if we do it this way. I think `int option option` is too much just for this.